### PR TITLE
Updated to Swift 2.1.1 & Improved UI

### PR DESCRIPTION
**Pull request by avazquezpr**
## Update

App is now displayed perfectly across all iPhone and iPad screen sizes. 
Updated code to Swift 2.1.1
## Screenshots

screen shot 2015-12-25 at 4 56 42 pm

### CODE REVIEW

**Comment by honghaoz** [comment]
@avazquezpr Great work! There are some conflicts need to be resolved. Can you fix it? Thanks for your PR!

**Comment by avazquezpr** [comment]
Sure, no problem! What are the conflicts?

**Comment by honghaoz** [comment]
screen shot 2016-01-06 at 1 34 28 pm
Did you pull from origin master? (My repo master branch). Since i cannot merge it

**Comment by avazquezpr** [comment]
Yes, i pulled from honghaoz/master. It must be because there are new commits that are not in my pull request's repo. Will work on that now :+1:

File: DynamicCollectionViewCellWithAutoLayout-Demo/ViewController.swift [hunk 1]
```diff
@@ -143,16 +147,5 @@ class ViewController: UIViewController, UICollectionViewDataSource, UICollection
...
-    // MARK: - Rotation
-    // iOS7
-    override func willRotateToInterfaceOrientation(toInterfaceOrientation: UIInterfaceOrientation, duration: NSTimeInterval) {
-        collectionView.collectionViewLayout.invalidateLayout()
-    }
-    
-    // iOS8
-    override func viewWillTransitionToSize(size: CGSize, withTransitionCoordinator coordinator: UIViewControllerTransitionCoordinator) {
-        collectionView.collectionViewLayout.invalidateLayout()
-    }
```
Comment (honghaoz) [inline DynamicCollectionViewCellWithAutoLayout-Demo/ViewController.swift:156]: why those methods are removed?

File: README.md [hunk 2]
```diff
@@ -2,12 +2,12 @@
 
 Demo for Collection View on iOS with auto layout in UICollectionViewCell to create cells with dynamic heights
 
-Works on iOS7 and iOS8
+Works on iOS7, iOS8 and iOS9.
 
 ## Updated
 
-For off screen cells, there's a new dynamic collection view which will handle it gracefully. Check it out!
-[ZHDynamicCollectionView](https://github.com/honghaoz/ZHDynamicCollectionView)
```
Comment (honghaoz) [inline README.md:10]: This shouldn't be removed.

**Comment by honghaoz** [comment]
everything else looks good :+1:

File: DynamicCollectionViewCellWithAutoLayout-Demo/ViewController.swift (hunk 1 above)
Comment (avazquezpr) [inline DynamicCollectionViewCellWithAutoLayout-Demo/ViewController.swift:156]: My mistake. Fixed.

**Comment by avazquezpr** [comment]
Fixed the conflicts :+1: Let me know if there is anything else i can help with!

**Comment by honghaoz** [comment]
:+1: Thanks @avazquezpr !

**Pull request merged**